cranelift/filetests/filetests/isa/aarch64/leaf_with_preserve_frame_pointers.clif

```diff
@@ -0,0 +1,18 @@
+;; Test compilation of leaf functions while preserving frame pointers.
```

Comment:
Can we add the equivalent of this "preserves frame pointers" test for x64 too? The flag won't actually matter now but it'd be good to have the test coverage if/when we add any omit-frame-pointers mode in the future.

Reply:
Done!